tests/envs/aifs.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
+set -eu
+set -o pipefail
+
+cat <<EOF
+Running aifs test in $BASEDIR/aifs/aifs-single
+
+EOF
+
+# Grab version of container branch
+export SCRIPT_DIR=$( cd -- "$( dirname -- "$(readlink -f "${BASH_SOURCE[0]}")" )" &> /dev/null && pwd )
+pushd $SCRIPT_DIR
+export VERSION="$(git symbolic-ref --short HEAD)"
+popd
+
+# Check out aifs scripts
+[[ -d "$BASEDIR/aifs" ]] || git clone https://git.nci.org.au/bom/cm/aifs_scripts.git "$BASEDIR/aifs"
+pushd "$BASEDIR/aifs/aifs-single"
+
+# Test for ICs preparation
+qsub << EOF
+#-------------------------------------------------------------------------------
+#PBS -P dx2
+#PBS -q copyq
+#PBS -l walltime=00:10:00,mem=10GB,ncpus=1
+#PBS -l jobfs=10GB
+#PBS -l storage=gdata/dx2+scratch/dx2+gdata/rt52+gdata/dk92+gdata/ux62+gdata/wr45
```

Comment:
Maybe
```suggestion
#PBS -l storage=gdata/dx2+scratch/dx2+gdata/rt52+gdata/dk92+gdata/ux62+gdata/wr45+scratch/$PROJECT
```
in case the container was not built in the dx2 project, same for the other storage flags

Reply:
Ah looks like you had that before but chose to remove it - in that case just document the environment should be built in the dx2 project at nci